server/pulp/server/webservices/views/plugins.py

```diff
@@ -29,4 +30,27 @@ class TypeResourceView(View):
 
 
 class TypesView(View):
-    pass
+    """
+    View for dealing with all plugin types.
+    """
+
+    @auth_required(authorization.READ)
+    def get(self, request, *args, **kwargs):
+        """
+        Get all type definitions
+
+        :param request: WSGI Request obect
+        :type  request: WSGIRequest
+        :return       : Serialized list of obects representing all available content types
```

Comment:
Style thing, but I generally don't align the colons vertically because then if one variable name changes more lines than necessary need to be modified. This will be less independent from a maintenance perspective. Not a required change.